x-pack/plugins/canvas/canvas_plugin_src/functions/browser/index.js

```diff
@@ -7,7 +7,7 @@
 import { functions as commonFunctions } from '../common';
 import { browser } from './browser';
 import { location } from './location';
-import { urlparam } from './urlparam';
 import { markdown } from './markdown';
+import { urlparam } from './urlparam';
 
-export const functions = [browser, location, urlparam, markdown, ...commonFunctions];
+export const functions = [browser, location, markdown, urlparam, ...commonFunctions];
```

Comment:
This is a no-op, I originally put timelion in here, and re-ordered alphabetically.